common/flatpak-dir-private.h

```diff
@@ -445,6 +445,8 @@ char *      flatpak_dir_get_origin (FlatpakDir   *self,
                                     GError      **error);
 GFile *     flatpak_dir_get_exports_dir (FlatpakDir *self);
 GFile *     flatpak_dir_get_removed_dir (FlatpakDir *self);
+GFile *     flatpak_dir_get_sideload_repos_dir (FlatpakDir *self);
+GFile *     flatpak_dir_get_runtime_sideload_repos_dir (FlatpakDir *self);
```

Comment:
I guess you'd have to add a wrapper API or something since `FlatpakDir` is not public. The use case would be both for USB OS updates and flatpak operations that go along with them. We'd use the system flatpak installation for both.